rfcs/text/0000-bundle-definition.md

```diff
@@ -0,0 +1,248 @@
+  secondary analysis bundles, and tertiary (meta-analysis or aggregate analysis) bundles. A new class of bundle could be
+  needed if a new analysis modality or data aggregation method were to be introduced.
+
+* As denoted in the bundle type notation sketch below, [RFC 7231](https://tools.ietf.org/html/rfc7231) parameters can be
+  used to designate the process that generated a bundle. A new class of process (for example, a new secondary analysis
+  pipeline) could cause a new `bundle-type` value with the same bundle type and subtype as previous secondary analysis
+  data bundles, but with a new parameter value for `hca-pipeline`.
+
+* Similarly, a new modality or format of primary experimental output data would cause a new primary data bundle
+  `bundle-type` value.
+
+### Conveying bundle types
+
+A new field, `bundle-type`, will be introduced to the DSS bundle manifest (the response to `GET bundle` in DSS). The
+field value is a string formatted in a similar manner
+to [DCP Media Types](https://docs.google.com/document/d/1TqihrgXjct9aDmTJO52_gE2WlpFysB1OkG9C8exmWTw) and consistent
+with the syntax defined in [RFC 7231](https://tools.ietf.org/html/rfc7231#section-3.1.1.1). DSS must require the bundle
+type to be specified when creating a new bundle, but should not enforce the vocabulary. DSS must allow subscriptions
+and search results to be predicated on bundle type using the same string predicates available with other metadata and
+manifest string fields.
+
+### Registry of Bundle Types
+
+Introduce a registry of bundle types to be maintained by the Data Store team with specification input from the Metadata
+team. The initial contents of the registry are:
+
+| Bundle Type              | Bundle Class        | Example `bundle-type` value                                 |
+|--------------------------|---------------------|-------------------------------------------------------------|
+| Project Metadata Bundle  | Utility bundle      | `hca/project`                                               |
+| Primary Sequence Bundle  | Primary data bundle | `hca/primary-data; hca-data-type:SmartSeq2`                 |
+| Primary Imaging Bundle   | Primary data bundle | `hca/primary-data; hca-data-type:sptx`                      |
+| Secondary Analysis Bundle| Analysis data bundle| `hca/analysis-output; hca-pipeline:snap-atac`               |
+| Reference Resource Bundle| Utility bundle      | `hca/analysis-reference; hca-resource-type:reference-genome`|
+| Expression Matrix Bundle | Analysis data bundle| `hca/analysis-output; hca-resource-type:expression-matrix`  |
+
+The registry would be maintained either as part of this git repository (HumanCellAtlas/dcp-community) or as part of a
+repository chosen by the Data Store team, subject to a pull request-based contribution model.
+
+### Acceptance Criteria [optional]
+
+*Acceptance criteria are the conditions that a RFC must satisfy to be accepted by users or other stakeholders.* 
```

Comment:
Looks like some template strings were included in the RFC text, here and below. Intentional?

Reply:
Thanks for flagging this. Some template strings are needed in the "Q&A" section for structure. I've removed the rest.